www_src/components/range/range.jsx

```diff
@@ -16,13 +16,16 @@ var Range = React.createClass({
       value: 100
     };
   },
+  onChange: function (e) {
+    this.valueLink.requestChange(parseFloat(e.target.value));
```

Comment:
No, since this is only tied to a UI update and we want that to happen as quick as possible. Immediate state changes get cached via a `cacheEdits` function, which is only sent to the api on a `pause` event